specifications/api/swagger.yaml

```diff
@@ -281,6 +281,22 @@ x-walletAccountPubkey: &walletAccountPubkey
   maxLength: 128
   example: 1423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db11423856bc91c49e928f6f30f4e8d665d53eb4ab6028bd0ac971809d514c92db1
 
+x-cosigner: &cosigner
+  type: string
+  pattern: "^(cosigner#)[0-9]*$"
```

Comment:
small note: the index is mandatory, so it should be `^(cosigner#)[0-9]+$`

Reply:
indeed, fixed